src/uint/boxed/rand.rs

```diff
@@ -43,7 +40,7 @@ impl RandomMod for BoxedUint {
     /// The variable-time nature of the algorithm should not pose a security issue so long as the
     /// underlying random number generator is truly a CSRNG, where previous outputs are unrelated to
     /// subsequent outputs and do not reveal information about the RNG's internal state.
-    fn random_mod(rng: &mut impl CryptoRngCore, modulus: &NonZero<Self>) -> Self {
+    fn random_mod(rng: &mut impl RngCore, modulus: &NonZero<Self>) -> Self {
```

Comment:
I would be very careful about this doc comment, as the user may not be aware of what constitutes a CSPRNG.

Reply:
At any rate, the top-line comment is no longer accurate.